app/commands/v2/publish.rb

```diff
@@ -154,7 +154,7 @@ def clear_published_items_of_same_locale_and_base_path
       def set_public_updated_at
         return if edition.public_updated_at.present?
 
-        if update_type == "major"
+        if update_type == "major" || previous_item.blank?
```

Comment:
I see that we also compare the `update_type` with `"major"` in a few other places in this command. Do those needs to be updated as well to do this?